code/modules/cm_marines/overwatch.dm

```diff
@@ -929,6 +975,202 @@
 	icon_state = "echodrop"
 	squad = SQUAD_MARINE_5
 
+//======UPP=======
+
+/obj/structure/supply_drop/upp1
+	icon_state = "alphadrop"
+	squad = SQUAD_UPP_1
+
+/obj/structure/supply_drop/upp2
+	icon_state = "bravodrop"
+	squad = SQUAD_UPP_2
+
+/obj/structure/supply_drop/upp3
+	icon_state = "charliedrop"
+	squad = SQUAD_UPP_3
+
+/obj/structure/supply_drop/upp4
+	icon_state = "deltadrop"
+	squad = SQUAD_UPP_4
+
+/obj/structure/machinery/computer/overwatch/upp/count_marines(list/data)
```

Comment:
what does this override actually change, just the job names that're being looked for?